src/ngraph/serializer.cpp

```diff
@@ -78,6 +78,7 @@
 #include "ngraph/op/fused/prelu.hpp"
 #include "ngraph/op/fused/scale_shift.hpp"
 #include "ngraph/op/fused/space_to_depth.hpp"
+#include "ngraph/op/fused/unsqueeze.hpp"
```

Comment:
Please add `case OP_TYPEID::Unsqueeze:` line between 2040 - 2081 lines and between 2161 - 2171 lines.